src/android/CHIPTool/app/src/main/res/layout/callback_item.xml

```diff
@@ -0,0 +1,44 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
`callback_item.xml` and `callback*` are pretty vague, can we replace with `cluster_callback_item.xml` and `clusterCallback*`? Same with `parameter_item.xml`, etc.